Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+@Library('corda-shared-build-pipeline-steps@5.1') _
+
+pipeline {
+    agent {
+        docker {
+            // Our custom docker image
+            image "build-zulu-openjdk:11"
+            label "standard"
+            registryUrl 'https://engineering-docker.software.r3.com/'
+            registryCredentialsId 'artifactory-credentials'
+        }
+    }
+    environment {
+        ARTIFACTORY_CREDENTIALS = credentials('artifactory-credentials')
+        CORDA_ARTIFACTORY_PASSWORD = "${env.ARTIFACTORY_CREDENTIALS_PSW}"
+        CORDA_ARTIFACTORY_USERNAME = "${env.ARTIFACTORY_CREDENTIALS_USR}"
+        HELM_CHART_REPO = "helm/charts"
+        HELM_REGISTRY = setHelmRegistry()
+        HELM_CHARTS_DIR = "helm-charts" // used in the helmPublisher method to set the directory the OCI should be located in.
+    }
+    options {
+        ansiColor('xterm')
+        buildDiscarder logRotator(daysToKeepStr: '14')
+        disableConcurrentBuilds()
+        timestamps()
+        timeout(activity: true, time: 10)
+    }
+    stages {
+        stage('Check Helm Changes') {
+            steps {
+                script {
+                    def charts = sh(returnStdout: true, script: "ls -d ${env.HELM_CHART_REPO}/*/").trim().split("\n")
```

Comment:
While i know this was copied from elsewhere, this seems a bit over complex for just publishing the charts in this repo.

Do we need the whole newCharts section? below this as this will be fairly stable / unchanging

Reply:
that's true, suppose i could simply set it to those charts as it will be unchanging, okay

Reply:
done